.github/workflows/continuous-integration.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: Install Dependencies
         run: npm install
       - name: Compile
-        run: yarn compile
+        run: npm run build
```

Comment:
Switch to npm intentional?

Reply:
Yes, since yarn is not being used for installing dependencies, I think it is confusing to still use both npm and yarn.